### ISSUE TITLE
hops-mixin: configureBuild within Mixin executed twice

### ISSUE THREAD

**Issue by saviomuc** [xing/hops#636]
## This is my intent (choose one)

- [X] I want to report a bug
- [ ] I want to request a feature or change
- [ ] I want to provide or change a feature

## The problem
I am not sure if it's a bug or not.

In my current project I wanted to extend the CSS config to also use a `sass-loader`. In order to see what the webpack config looked like (after all presets have been applied) I created a `mixin.core.js` file to log `webpackConfig` within `configureBuild`. When logging I found out that `configureBuild` seems to be executed twice.

This can be reproduced with this code:

```javascript
// mixin.core.js
let count = 0;

class WebpackMixin extends Mixin {
  configureBuild(webpackConfig, target) {
    switch (target) {
      case 'build':
        break;
      case 'develop':
        count++;
        console.log('Show allLoader', count, webpackConfig);
        break;
      case 'node':
        break;
      default:
        break;
    }

    return webpackConfig;
  }
}

module.exports = WebpackMixin;
```

I am using these dependencies:
```
    "hops-graphql": "11.0.0-rc.28",
    "hops-mixin": "^11.0.0-rc.25",
    "hops-postcss": "11.0.0-rc.28",
    "hops-preset-defaults": "11.0.0-rc.27",
    "hops-react": "11.0.0-rc.27",
```

## Proposed solution
configureBuild should only be executed once.

**Comment by ZauberNerd** [comment]
I'm not near a computer right now, but the hook `configureBuild` would be
executed twice, once for the target `node` and once for the target
`develop` (or `build`).

Also: The `configureBuild` hook has three arguments `configureBuild(config,
loaders, target)`, so you should `switch` over the third argument.

I'll look into this tomorrow and try to reproduce.

Another hint: We have some debug output via the `debug` module.
You can see that by setting the `DEBUG` environment variable to
`DEBUG=hops*,untool* hops start`

Hope that helps for now ;)

**Comment by saviomuc** [comment]
@robin-drexler Actually it didn't cause any troubles. Was just a minor inconvenience and thought was worth reporting =)

**Comment by ZauberNerd** [comment]
This has landed in Hops master via 126eeda and will be available with the next release.
